### PR TITLE
portfolio: details popup window

### PR DESCRIPTION
**Pull request by X-Elie-X**
## In this PR we implemented the below:

- Creating a new repository.
- Upload all my portfolio files.
- we Created a popup window.
- Add main.js to activate the buttons and show the `popup` effects.
- Add some style for the `popup` design

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+}
+close.addEventListener('click', closePopup);
+
+// function on() {
+//   document.getElementById('#overlay').style.display = 'bloack';
+// }
+
+// function off() {
+//   document.getElementById('#overlay').style.display = 'none';
+// }
```
Comment (codecaiine) [inline main.js:36]: - [ ] I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile:

File: main.js [hunk 2]
```diff
@@ -1,13 +1,36 @@
+const listInfo = document.getElementById('listinfo');
```
Comment (codecaiine) [inline main.js:1]: - [ ] Please create an array of objects to store JavaScript objects containing the information for each project in your JavaScript file :+1: 

- [ ] Make sure that your project object contains information about the project: `name`, `description`, `image`, `technologies`, `links to live version`, and `link to the project source code`.
you should have something like that: 
```javascript
const projects = [{
    name: "project name",
    description: "Build portfolio popup window",
    image: "link_to_image",
    technologies: ["html", "JavaScript", "Css"],
    link: "link_to_live_version.com",
}]
```

File: index.html [hunk 3]
```diff
@@ -420,6 +436,59 @@ <h3>.</h3>
...
+        </button>
+        <button
+          type="button"
+          class="popup-btn effect"
+          onclick="document.location='https://github.com/X-Elie-X'"
+        >
+          See Source<img src="./img/seesourcce.png" alt="seesourcce" />
+        </button>
+      </div>
+    </section>
```
Comment (codecaiine) [inline index.html:491]: - [ ] Please ensure that your projects section is created dynamically with the information from the projects array you created in your Javascript file :+1:  

- [ ] Please make sure that your modal popup data is populated with the data from the project's array :smile:

File: index.html [hunk 4]
```diff
@@ -420,6 +436,59 @@ <h3>.</h3>
       </li>
     </ul>
     <hr class="hr3" />
+    <!-- popoup section -->
+    <section class="popup" id="popup1">
```
Comment (codecaiine) [inline index.html:440]: - [ ] Good job so far your popup is displaying :+1: But it is not working as expected :cry:  For Desktop your popup does not conform to the dimensions set in the Figma template. You can see a difference when you compare it to Figma. Please use the dimensions and follow the layout as defined in the Figma template to make them similar. :smile: 

### Figma template
Screenshot from 2022-07-20 09-53-21

### Your project
Screenshot from 2022-07-20 09-51-46




- [ ] For the Mobile version the details about the project do not appear. The elements are improperly placed as you can see in the picture below :disappointed: Please fix it so they can appear on the popup so it looks similar to the Figma template. :+1: 

### Figma template
Screenshot from 2022-07-20 10-31-46

### Your work
Screenshot from 2022-07-20 09-54-24

File: index.html [hunk 5]
```diff
@@ -420,6 +436,59 @@ <h3>.</h3>
       </li>
     </ul>
     <hr class="hr3" />
+    <!-- popoup section -->
+    <section class="popup" id="popup1">
+      <button type="button" class="close-popup" id="close-popup">X</button>
```
Comment (codecaiine) [inline index.html:441]: - [ ] We can not click or tap the **close (X) button,** and the popup disappear. :disappointed:   Please fix it to give your user a great user experience :smile:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @TEAM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
No linters errors :heavy_check_mark: 
Good pull request description :heavy_check_mark: 

## Required Changes ♻️

- [x] Good job so far but Please [Check the project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) because some requirements are not implemented well :+1: 
- [ ] I would like to recommend you to work on your mobile version because it s not working as expected , you need to give to your user a good experience :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @X-Elie-X , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well described 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**